templates/cluster-template-external-loadbalancer.yaml

```diff
@@ -60,6 +60,7 @@ spec:
       storagePolicyName: '${VSPHERE_STORAGE_POLICY}'
       template: '${VSPHERE_TEMPLATE}'
       thumbprint: '${VSPHERE_TLS_THUMBPRINT}'
```

Comment:
I would have to look up the details but I think we have to modify flavorgen and then re-generate those files

Reply:
Related: https://github.com/kubernetes-sigs/cluster-api-provider-vsphere/issues/2153